Packs/DSPM/Integrations/DSPM/DSPM.py

```diff
@@ -0,0 +1,996 @@
+        "categoryType.equals": args.get("categoryTypeEquals"),
+        "status.in": args.get("statusIn"),
+        "status.equals": args.get("statusEquals"),
+        "sort": args.get("sort"),
+        "page": page,
+        "size": MAX_PAGE_SIZE
+    }
+
+    # Remove None values from params
+    params = {k: v for k, v in params.items() if v is not None}
+
+    alerts_list = client.get_alerts_list(params)
+    return alerts_list
+
+
+def update_dspm_alert_status(client, args):
+    alert_id = args.get("alertId")
+    status = args.get("status")
+    if status and status not in SUPPORTED_STATUSES:
+        raise ValueError(f'This "{status}" status is not supported')
+
+    try:
+        response = client.update_alert_status(alert_id, status)
+        # Format the response for display
+        headers = ["Alert ID", "Old Status", "New Status"]
+        data = {
+            "Alert ID": response.get("alertId"),
+            "Old Status": response.get("oldStatus"),
+            "New Status": response.get("newStatus"),
+        }
+
+        markdown = tableToMarkdown("Alert Status Update", [data], headers=headers, headerTransform=pascalToSpace)
+
+        return CommandResults(
+            readable_output=markdown,
+            outputs_prefix="DSPM.AlertStatusUpdate",
+            outputs_key_field="alertId",
+            outputs=data,
+        )
+    except Exception as e:
+        return_error(
+            f"Failed to update alert '{alert_id}' to status '{status}'. Error: {str(e)}"
+        )
+
+
+def get_integration_config():
+
+    integration_config = {
+        "dspmApiKey": demisto.params().get("dspmApiKey", {}).get("password"),
+        "slackMsgLifetime": demisto.params().get("slackMsgLifetime"),
+        "defaultSlackUser": demisto.params().get("defaultSlackUser"),
+    }
+    demisto.debug(f" integration config : ${integration_config}")
+
+    # Prepare data for table format
+    table_data = [
+        {"Key": key, "Value": value} for key, value in integration_config.items()
+    ]
+
+    # Convert dictionary to table format
+    markdown = tableToMarkdown(
+        "Integration Configuration", table_data, headers=["Key", "Value"]
+    )
+
+    # Return CommandResults with the integration config
+    return CommandResults(
+        readable_output=markdown,
+        outputs_prefix="DSPM.IntegrationConfig",
+        outputs_key_field="config",
+        outputs={"integration_config": integration_config},
+    )
+
+
+def get_list_of_labels(client: Client):
+    """Command to fetch list of label names based on company"""
+    labels = client.get_labels()
+    labels_formatted = [
+        {"No": index + 1, "Key": dt} for index, dt in enumerate(labels)
+    ]
+
+    table_name = "Labels"
+    headers = ['No', 'Key']
+    if labels_formatted:
+        readable_output = tableToMarkdown(table_name, labels_formatted, headers=headers)
+    else:
+        readable_output = tableToMarkdown(table_name, "No data found", headers=headers)
+
```

Comment:
```suggestion
    readable_output = tableToMarkdown(table_name, labels_formatted, headers=headers)
```

Reply:
We will make required changes in the rest of the commands as well.